src/rebar_relx.erl

```diff
@@ -72,13 +71,42 @@ do(Provider, State) ->
 
     {ok, State}.
 
+read_relx_config(State, Options) ->
+    ConfigFile = proplists:get_value(config, Options, []),
+    case ConfigFile of
+        "" ->
+            case {rebar_state:get(State, relx, []), file:consult("relx.config")} of
+                {[], {ok, Config}} ->
+                    ?DEBUG("Using relx.config", []),
```

Comment:
`Configuring releases with relx.config`

Reply:
Fixed in https://github.com/erlang/rebar3/pull/2435/commits/96d61cf5f80a8917b66c73eb41a8ec4fd1773f2e